panoramapublic/src/org/labkey/panoramapublic/PanoramaPublicController.java

```diff
@@ -711,16 +712,16 @@ public ModelAndView getView(JournalSupportContainerForm form, boolean reshow, Bi
             }
 
             VBox view = new VBox();
-            if(errors.getErrorCount() > 0)
+            if (errors.getErrorCount() > 0)
             {
-                StringBuilder html = new StringBuilder();
+                HtmlStringBuilder html = HtmlStringBuilder.of();
                 for(ObjectError error: errors.getAllErrors())
                 {
                     HtmlString errHtml = createHtmlFragment(SPAN(cl("labkey-error"), error.getDefaultMessage()));
-                    errHtml.appendTo(html);
+                    html.append(errHtml);
                 }
-                BR().appendTo(html);
-                view.addView(new HtmlView(HtmlString.unsafe(html.toString())));
+                html.append(createHtml(BR()));
+                view.addView(new HtmlView(html));
```

Comment:
I think lines 717 to 724 can be replaced with `view.addView(new HtmlView(ERRORS(errors)))`

Reply:
Yes! Thanks. I forgot this was a DOM.LK method.